python/GangaLHCb/Lib/Applications/GaudiExec.py

```diff
@@ -205,6 +206,10 @@ def prepare(self, force=False):
         self.configure(self)
         logger.info('Preparing %s application.' % getName(self))
 
+	# Make sure the summary.xml file gets downloaded for the metadata
+        if self.getMetadata:
+            self.getJobObject().outputfiles.append(LocalFile('summary.xml'))
```

Comment:
Are we writing into the user controlled `outputfiles` attribute here? This will potentially cause trouble when the job is copied. Could lead to either that it is added twice or that it is there even if the user has turned off the `getMetadata` flag.